pages/linux/vnstati.md

```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> PNG image output support for vnStat.
+> More information: <https://manned.org/vnstati>.
+
+- Output a summary of the last 2: months, days, and all-time:
+
+`vnstati --summary --iface {{network_interface}} --output {{output.png}}`
+
+- Output all-time top 10 traffic days:
+
+`vnstati --top10 --iface {{network_interface}} --output {{output.png}}`
+
+- Output traffic statistics on a monthly basis for the last 12 months:
```

Comment:
```suggestion
- Output traffic statistics for every month for the last 12 months:
```

Leave this suggestions open for others to see

Reply:
```suggestion 

- Output traffic statistics for each month, for the last 12 months:

Reply:
```suggestion 
- Output traffic statistics for every month, for the last 12 months:
```

Reply:
Just another one:
```suggestion
- Output monthly traffic statistics from the last 12 months:
```

Reply:
@melvin2204 just apply the one you like the most.